psi4/src/read_options.cc

```diff
@@ -2502,11 +2502,15 @@ int read_options(const std::string &name, Options &options, bool suppress_printi
         options.add_str("DLPNO_LOCAL_ORBITALS", "BOYS", "BOYS PIPEK_MEZEY");
         /*- Maximum number of iterations to determine the MP2 amplitudes. -*/
         options.add_int("DLPNO_MAXITER", 50);
+        /*- Perform DLPNO-CCSD? */
```

Comment:
```suggestion
        /*- Perform DLPNO-CCSD? !expert*/
```
user shouldn't set this, only the driver for internal routing